CDCgov/prime-reportstream
# Updates baseUrl to include cache query by current date as ISOString

This PR changes the React app to use the cache query when getting reports.

Test Steps:
1. pull `kevinhaube/2784-cached-reports-call
2. `yarn` and `yarn start`
3. navigate to the reports page and ensure things load properly
4. press F12 to open Dev tools > Network
5. Ensure call is using query for cache and returning 200 w/ a body

## Changes
![Screenshot from 2021-10-28 15-26-24](https://user-images.githubusercontent.com/4022304/139322724-ce3fee7b-1c3a-4ca0-a534-37513068f59e.png)

## Checklist

### Testing

### Process

## Fixes
- #2784 

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
No
- Does this change introduce new dependencies that need vetting?
No
- Does this change require changes to our infrastructure?
No
- Does logging contain sensitive data?
No
- Does this PR include or remove any sensitive information itself?
No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
